### DISCUSSION TITLE
Edit of webradio is impossible if scrolled down

### DISCUSSION THREAD

**Discussion by AndyFx12** [rern/rAudio#1751] [Bug Reports]
Hi rern,

if webradio-page is scrolled down, editing a webradio shows a truncated window:

![grafik](https://github.com/user-attachments/assets/72fbd999-942c-4570-ac7f-e36ffd9c8c8a)

So, editing is impossible.

regards, Andy

**Comment by hjfisch** [comment]
It is a bit tricky , but it is possible . You need to make the main-window in your browser with strg + mouse-wheel a bit smaller if you have a big radio-list . Than you can edit your stations . After this is done , make it bigger and better for you to read again . Doing this with your smartphone is impossible . You need to use your PC or Mac .

**Comment by rern** [comment]
Please try the UPDATE again.

**Reply by AndyFx12** [reply]
It works fine, now. Thanks!
